packages/vendor-core/package.json

```diff
@@ -24,13 +24,13 @@
   "dependencies": {
     "@apollo/client": "^3.3.7",
     "@novnc/novnc": "1.3",
-    "@patternfly/patternfly": "^4.171.1",
-    "@patternfly/react-charts": "~6.94.15",
-    "@patternfly/react-core": "~4.271.2",
-    "@patternfly/react-icons": "~4.93.3",
-    "@patternfly/react-styles": "~4.92.3",
-    "@patternfly/react-table": "~4.112.6",
-    "@patternfly/react-tokens": "~4.94.3",
+    "@patternfly/patternfly": "^5.2.0",
+    "@patternfly/react-charts": "^5.2.0",
```

Comment:
Should be [7.3.1](https://www.npmjs.com/package/@patternfly/react-charts/v/7.3.1)